merlin/main.py

```diff
@@ -755,6 +763,13 @@ def generate_worker_touching_parsers(subparsers: ArgumentParser) -> None:
         help="Specify desired Merlin variable values to override those found in the specification. Space-delimited. "
         "Example: '--vars LEARN=path/to/new_learn.py EPOCHS=3'",
     )
+    run_workers.add_argument(
+        "--disable-logs",
+        action="store_true",
+        default=False,
```

Comment:
Remove the default=False line.  It’s not necessary and could perhaps lead to bugs:
https://stackoverflow.com/questions/70428108/whats-the-point-of-having-both-action-store-true-and-default-false-in-parser